precompiles/referenda/src/lib.rs

```diff
@@ -49,6 +49,90 @@ type BoundedCallOf<Runtime> = Bounded<<Runtime as pallet_referenda::Config>::Run
 type OriginOf<Runtime> =
 	<<Runtime as frame_system::Config>::RuntimeOrigin as OriginTrait>::PalletsOrigin;
 
+pub struct TrackInfo {
+	name: UnboundedBytes,
+	max_deciding: U256,
+	decision_deposit: U256,
+	prepare_period: U256,
+	decision_period: U256,
+	confirm_period: U256,
+	min_enactment_period: U256,
+	min_approval: UnboundedBytes,
+	min_support: UnboundedBytes,
+}
+
+impl EvmData for TrackInfo {
```

Comment:
Could this be done as a derive macro?

Reply:
We can add an EvmData derive macro in a follow up PR yes :)